chainerx_cc/chainerx/python/CMakeLists.txt

```diff
@@ -21,17 +29,16 @@ pybind11_add_module(_core.so MODULE
     testing/device_buffer.cc
     testing/testing_module.cc
     )
-
 if(${APPLE})
-    target_link_libraries(_core.so PRIVATE "-Wl,-rpath,@loader_path" chainerx)
+    target_link_libraries(${module_name} PRIVATE "-Wl,-rpath,@loader_path" chainerx)
 else()
-    target_link_libraries(_core.so PRIVATE "-Wl,-R,'$ORIGIN/.'" chainerx)
+    target_link_libraries(${module_name} PRIVATE "-Wl,-R,'$ORIGIN/.'" chainerx)
 endif()
 
 # Visibility (CXX_VISIBILITY_PRESET) must be set to "default" to register custom exceptions, overriding the visibility configured by pybind11_add_module.
 # Note however that this only seems to be an issue when building with libc++.
-set_target_properties(_core.so
+set_target_properties(${module_name}
     PROPERTIES
     PREFIX "${PYTHON_MODULE_PREFIX}"
     SUFFIX "${PYTHON_MODULE_SUFFIX}"
-    CXX_VISIBILITY_PRESET "default")
+    CXX_VISIBILITY_PRESET "default")
```

Comment:
Could you place an end-of-line?

Reply:
fixed it
https://github.com/durswd/chainer/commit/12a0799e889c60a90c9a8e428bb9b709e67ee9bd